src/windows/staff.c

```diff
@@ -593,12 +593,12 @@ void window_staff_overview_dropdown(rct_window *w, int widgetIndex, int dropdown
  */
 void window_staff_overview_update(rct_window* w)
 {
-	int var_496 = RCT2_GLOBAL((int)w + 0x496, uint16);
+	int var_496 = RCT2_GLOBAL((uintptr_t)w + 0x496, uint16);
```

Comment:
Can you just modify the window struct to add var_496. It looks to be some sort of animation frame count.